src/protocols/mqtt5/Mqtt5Protocol.cpp

```diff
@@ -548,7 +545,7 @@ void Mqtt5Protocol::sendConnAck(const IStreamConnectionPtr& connection, const Mq
 {
     unsigned int sizePropPayload = 0;
     unsigned int sizePayload = Mqtt5Serialization::sizeConnAck(data, sizePropPayload);
-    int sizeMessage = 1 + Mqtt5Serialization::sizeVarByteNumber(sizePayload) + sizePayload;
+    unsigned int sizeMessage = 1u + Mqtt5Serialization::sizeVarByteNumber(sizePayload) + sizePayload;
```

Comment:
size cant be negative right. I think we can even make it size_t or?? @beckdave

Reply:
When I use signed, then I am afraid that the value could be involved in a substraction where the result could be negative. In case of unsigned the result would a very big number. To avoid this postential situation, I often use signed. But in this case, I guess it is ok to use unsigned.